### DISCUSSION TITLE
Moving recordings to NAS or external drive?

### DISCUSSION THREAD

**Discussion by mcdent** [blakeblackshear/frigate#2566] [Ask A Question]
Hey, wondering if anybody has a script for safely moving recordings to an external drive or NAS ?

I'm finding my 1TB SSD is filling up fairly fast with 6 cameras. I keep lowering the retention settings but would prefer not too.

I'd probably connect and old/slow external hard drive via USB2 but dont really want to use that for recordings due to it being slooow.

Thanks!

**Comment by NickM-27** [comment]
What are you running this on? Could definitely set up a script to just do this automatically periodically

**Reply by mcdent** [reply]
Yeah cheers! I'm running it on a regular home assistant install on Generic x86-64 (intel nuc type box)

**Comment by alexelite** [comment]
I also use the biggest drive I have, 1TB, for recordings. With 4 cameras (4MP) recording continuously, it fills up in 4 days. 
I used Blue Iris before giving Frigate a try and I like BI's approach on storage settings. Users can define multiple storage locations with size and time limits, and also rules on what to do when limits are reached.  Cameras can individually be assigned to storage locations. 
In my setup I used 2 HDD drives, 2 cameras on each drive.

Because Frigate uses /YEAR-MONTH/DAY/HOUR/{camera}/*.mp4 I could not think of a way (like using symlinks) to achieve similar results.
